src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -122,6 +123,14 @@ public ResponseEntity<Page<MeasureListDTO>> getMeasures(
     return ResponseEntity.ok(measures);
   }
 
+  @GetMapping("/measures/count")
+  public ResponseEntity<Map> getCounts(Principal principal) {
```

Comment:
Can we be strick on typing ?
```suggestion
  public ResponseEntity<Map<String, Integer>> getCounts(Principal principal) {
```

Reply:
Fixed